src/asciidoc/docs/getting-started/developer-updates/RecentUpdates.adoc

```diff
@@ -5,62 +5,59 @@
 This digest summarizes the recent updates to the Koji platform, packages, and developer site.
 For highlights from earlier updates, see <<developer-updates#>>.
 
-include::../../_includes/tip-discord-server.adoc[tags=*]
-
 // tag::highlights[]
-=== Developer site updates
 
-* To accelerate Koji app development, several articles have been updated, including the following:
-** Updated – <<introduction#>>, <<projects#>>, <<editor#>>, <<local-development#>>, <<port-existing-code#>>
-** New – <<entitlements#>>
+=== Core package updates
 
-* To make it easier to find the content you need, the site navigation has been streamlined and some deprecated content has been removed.
+* The following contexts are deprecated: `admin`, `about`, and `sticker`. Apps can provide admin interfaces by managing roles with Identity.
 
-=== Koji platform updates
+* The `estimatePoses` option is no longer supported for video captures (Koji.ui.capture.video). Apps can process camera input directly, if needed.
 
-* The platform navigation has been updated to support the Koji Profile as a central home for a creator's Koji apps.
-Related changes include:
+* Bug fixes
 
-** The Koji button has been replaced with a new floating menu on profiles and a top navigation bar on app.
-This new navigation is less invasive and brings the experience of managing a profile or app much close to the app itself.
+Learn more: https://github.com/madewithkoji/koji-core/releases[Koji Core release notes]
 
-** Creators can no longer remix apps directly.
-Instead, when viewing someone else's app, creators can tap the app name to view the App Store listing and preview or use the app from there.
-+
-image::appdetails-jun2021.png[App Store listing]
+=== Koji platform updates
 
-* You can now make withdrawals from your Koji account without any fees, so you can retain more revenue from your apps.
-// end::highlights[]
+* During app customization, `remix.encryptValue` and `remix.decryptValue` can now read and write values from the app's keystore.
```

Comment:
This update applies specifically to editing an app that has already been published. New customizations have always been able to perform these actions.

Reply:
Fixed.
BEFORE:  During app customization, ....
AFTER: When you edit an app that has already been published, ....

Thanks much!